doc/Overview/Logging/LoggingOverview.md

```diff
@@ -1,108 +1,182 @@
+
+| Level | Description |
+|-------|-------------|
+| Trace | Used for parts of a method to capture a flow. |
+| Debug | Used for diagnostics information. |
+| Information | Used for general successful information. Generally the default minimum. |
+| Warning | Used for anything that can potentially cause application oddities. Automatically recoverable. |
+| Error | Used for anything that is fatal to the current operation but not to the whole process. Potentially recoverable. |
+| Critical | Used for anything that is forcing a shutdown to prevent data loss or corruption. Not recoverable. |
+
+Because these extension methods are scoped to a specific log level, the standard `Log()` method on the `ILogger<T>` interface does not need to be used.
+
+## Configuring Logging Output
+
+The `UseLogging()` extension method accepts an optional `Action<ILoggingBuilder>` parameter that can be used to configure the logging output.
+
+### Setting the Minimum Log Level
+
+The minimum log level can be set by calling the `SetMinimumLevel()` extension method on the `ILoggingBuilder` instance. If no LogLevel is specified, logging defaults to the `Information` level which may not always be desirable. The following example shows how to change the minimum log level to `Debug`:
 
-public App()
+```csharp
+protected override void OnLaunched(LaunchActivatedEventArgs e)
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseLogging()
-        .UseSerilog()
-        .Build();
-    // ........ //
-}
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host.UseLogging(
+                builder => {
+                    builder.SetMinimumLevel(LogLevel.Debug);
+                });
+        });
+...
 ```
 
-## Logging
+### Setting the XAML Log Level
 
-To log, you simply need to get a `ILogger` and use the appropriate methods.
+Sometimes it's necessary to filter messages recorded for specific XAML types to reduce noise. When a preference is specified for the **XAML log level**, the logger will change the verbosity of events from a set of specific XAML-related namespaces and types:
+- Microsoft.UI.Xaml
+- Microsoft.UI.Xaml.VisualStateGroup
+- Microsoft.UI.Xaml.StateTriggerBase
+- Microsoft.UI.Xaml.UIElement
+- Microsoft.UI.Xaml.FrameworkElement
+
+It can be set by calling the `XamlLogLevel()` extension method on the `ILoggingBuilder` instance. The following example shows how to set the XAML log level to `Information`:
 
 ```csharp
-var myLogger = myServiceProvider.GetService<ILogger<MyType>>();
+protected override void OnLaunched(LaunchActivatedEventArgs e)
+{
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host.UseLogging(
+                builder => {
+                    builder.XamlLogLevel(LogLevel.Information);
+                });
+        });
+...
+```
 
-myLogger.LogInformation("This is an information log.");
+### Setting the Layout Log Level
```

Comment:
I don't think we need to go into detail about the various extension methods that can control logging for different groups of namespaces. Perhaps just provide a summary of these methods under the internal logging section